src/Estimators/WalkerLogInput.h

```diff
@@ -0,0 +1,46 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2024 QMCPACK developers.
+//
+// File developed by: Jaron T. Krogel, krogeljt@ornl.gov, Oak Ridge National Laboratory
+//
+//////////////////////////////////////////////////////////////////////////////////////
+#ifndef QMCPLUSPLUS_WALKERLOGINPUT_H
+#define QMCPLUSPLUS_WALKERLOGINPUT_H
+
+#include "InputSection.h"
+
+namespace qmcplusplus
+{
+
+/** Native representation for walker logs input
+ */
+struct WalkerLogInput : public InputSection
+{
+  bool present;
```

Comment:
@PDoakORNL should this be a base class feature?

Reply:
no